x-pack/plugins/ml/server/models/results_service/results_service.ts

```diff
@@ -57,7 +57,8 @@ export function resultsServiceProvider(client: IScopedClusterClient) {
     dateFormatTz: string,
     maxRecords: number = ANOMALIES_TABLE_DEFAULT_QUERY_SIZE,
     maxExamples: number = DEFAULT_MAX_EXAMPLES,
-    influencersFilterQuery: any
+    influencersFilterQuery?: any,
+    actualPlotFunction?: string | undefined
```

Comment:
Maybe this should be renamed to `functionDescription` to match the name of the property in the records?

Reply:
Updated here https://github.com/elastic/kibana/pull/81662/commits/cb2429e192079f4e9a1cfc0e401a2a9544d19f23